### PR TITLE
add libopus to conda-forge

### PR DESCRIPTION
**Pull request by wolfv**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/libopus/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    - autoconf           # [unix]
+    - libtool            # [unix]
+    - automake           # [unix]
+    - ninja              # [win]
+    - cmake              # [win]
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libopus.so             # [linux]
+    - test -f ${PREFIX}/lib/libopus.dylib          # [osx]
```
Comment (chrisburr) [inline recipes/libopus/meta.yaml:32]:
```suggestion
    - test -f ${PREFIX}/lib/libopus${SH_EXT}             # [unix]
```

File: recipes/libopus/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    - conda inspect linkages -p $PREFIX libopus   # [not win]
+    - conda inspect objects -p $PREFIX libopus    # [osx]
+
+about:
+  home: http://opus-codec.org/
+  license: 3-clause BSD
+  license_family: BSD
```
Comment (chrisburr) [inline recipes/libopus/meta.yaml:39]:
```suggestion
  license: BSD-3-Clause
```

File: recipes/libopus/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    - automake           # [unix]
+    - ninja              # [win]
+    - cmake              # [win]
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libopus.so             # [linux]
+    - test -f ${PREFIX}/lib/libopus.dylib          # [osx]
+    - conda inspect linkages -p $PREFIX libopus   # [not win]
+    - conda inspect objects -p $PREFIX libopus    # [osx]
```
Comment (chrisburr) [inline recipes/libopus/meta.yaml:34]:
```suggestion
```

File: recipes/libopus/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  name: libopus
+  version: {{ version }}
+
+source:
+  url: https://github.com/xiph/opus/archive/v{{ version }}.tar.gz
+  sha256: 4834a8944c33a7ecab5cad9454eeabe4680ca1842cb8f5a2437572dbf636de8f
+
+build:
+  number: 0
+  skip: True  # [win]
```
Comment (chrisburr) [inline recipes/libopus/meta.yaml:13]:
```suggestion
```

**Comment by wolfv** [comment]
Ok, I am done with this :) let me know what you think @chrisburr

**Pull request merged**